Wiz/wiz/wiz_issues_connector.py

```diff
@@ -0,0 +1,177 @@
+import asyncio
+import time
+from datetime import datetime, timedelta, timezone
+from typing import Any, Optional
+
+import orjson
+from dateutil.parser import isoparse
+from loguru import logger
+from sekoia_automation.aio.connector import AsyncConnector
+from sekoia_automation.connector import DefaultConnectorConfiguration
+from sekoia_automation.storage import PersistentJSON
+
+from wiz import WizModule
+from wiz.client.gql_client import WizErrors, WizGqlClient
+from wiz.metrics import EVENTS_LAG, FORWARD_EVENTS_DURATION, OUTCOMING_EVENTS
+
+
+class WizIssuesConnectorConfig(DefaultConnectorConfiguration):
+    """WizIssuesConnector configuration."""
+
+    frequency: int = 60
+
+
+class WizIssuesConnector(AsyncConnector):
```

Comment:
In order to ease the development of next connectors for Wiz, can you create a base connector holding common stuff and create a Wiz Issue connector that inherits from the base connector?
It will allow us to create the next connectors by inheriting from this base.